app/main.py

```diff
@@ -1,2 +1,56 @@
+import math
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self._initialize_hash_table(8)
+        self.data = []
+        self.capacity = 0
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        self.data.append([key, value])
+        self.create_hash_table(key, value)
+        self.capacity += 1
+        load_factor = 2 / 3
```

Comment:
Add a `LOAD_FACTOR = 2/3` at the beginning of a file to indicate that it is a constant, currently, it is lost for quick change in the  `__setitem__` method.